### ISSUE TITLE
Blacklisted Artists

### ISSUE THREAD

**Issue by rdricco** [InputUsername/rescrobbled#39]
Can you help me to create a script to blacklist some artists? I don't know but maybe this one can be a nice built-in feature.

**Comment by InputUsername** [comment]
Thanks for the suggestion, that would definitely be a useful feature. Currently, if the filter script doesn't return artist, title and album (ie. one or more are missing), the original artist/title/album are used.

I could change this into not submitting songs where the filtered info is missing. That way the filter script could filter songs by outputting nothing. Python example:
```python
ignored_artists = {'Queen', 'Justin Bieber'}

artist, title, album = (l.rstrip() for l in sys.stdin.readlines())

if artist not in ignored_artists:
    print(artist, title, album, sep='\n')
```

Does that sound reasonable to you?

**Comment by rdricco** [comment]
Looks great, I think will work fine.

**Comment by InputUsername** [comment]
@rdricco I've just released [`v0.4.0`](https://github.com/InputUsername/rescrobbled/releases/tag/v0.4.0) which includes this functionality :)
